### PR TITLE
Uma skills

### PR DESCRIPTION
**Pull request by manucamejo**

### CODE REVIEW

File: server/lib/dark_worlds_server/engine/runner.ex [hunk 1]
```diff
@@ -82,6 +82,7 @@ defmodule DarkWorldsServer.Engine.Runner do
     Process.send_after(self(), :all_characters_set?, @character_selection_check_ms)
     Process.send_after(self(), :character_selection_time_out, @character_selection_timeout_ms)
     Process.send_after(self(), :check_player_amount, @player_check)
+    Process.send_after(self(), :shrink_map, @map_shrink_wait_ms)
```
Comment (tvillegas98) [inline server/lib/dark_worlds_server/engine/runner.ex:85]: Remove this

**Review by tvillegas98 — CHANGES_REQUESTED** [review]

File: server/lib/dark_worlds_server/engine/runner.ex (hunk 1 above)
Comment (manucamejo) [reply server/lib/dark_worlds_server/engine/runner.ex]: Done

**Review by ncontinanza — CHANGES_REQUESTED** [review]
There's an overlapping of the poison ability with the mark, in a way that the damage due to the poison is never being applied.

**Review by tvillegas98 — APPROVED** [review]
LGTM!

**Pull request merged**